content/_index.md

```diff
@@ -1,91 +1,154 @@
+verfasst von zwei Ehrenamtlichen unserer Community.
+
+<div class="landing-callout">
+In der Blogserie <a href="/blog">Out in the Open</a> analysieren unsere
+Expert*innen aktuelle Ereignisse zu Digitalpolitik, Open Data und
+Civic Tech in Deutschland.
+</div>
 
+### Politik
 
-{{< section >}}
-{{< headline-icon icon="icon-documents.svg" >}}
+Entscheidungsträger*innen in der Politik müssen sich in Gesetzgebungsprozessen für Transparenz, Offene Daten und Open-Government-Strategien einsetzen. Damit stärken sie demokratische Verfahren, Wissensbildung und Partizipation. Zusätzlich schaffen sie Vertrauen in der Öffentlichkeit und legitimieren das eigenen Handeln. Politiker*innen können sich für langfristige Förderungen einsetzen, die digitales Ehrenamt und IT-Infrastruktur in Deutschland stärken.
 
-{{< headline-brackets-h2 color="blue"  >}}
-Projekte und Anwendungen finden
-{{< /headline-brackets-h2  >}}
+Von den Expert*innen von Code for Germany erfahren Politiker*innen mehr über nachhaltig wirksame digitale Lösungsansätze für Herausforderungen in ihren Wahlkreisen und stärken den Dialog mit Bewohner*innen. Interessierte Politiker*innen informieren sich in der Open Data Knowledgebase, kontaktieren ihre lokale Community oder schreiben uns eine Nachricht.
 
-{{< paragraph-center class="text-center mb-5" >}}
-Du möchtest einen Eindruck von unseren Projekten bekommen? Wir arbeiten gegenwärtig zu den Themenfeldern Mobilität, Gesellschaft, Umwelt und Politik.
-{{< /paragraph-center >}}
+### Unternehmen
 
-{{< button-center href="/projekte" >}}
-Projekte
-{{< /button-center >}}
-{{< /section >}}
+Unternehmen können selbst Offene Daten publizieren, Open Source Software verwenden und eigenen Source Code kostenfrei bereitstellen. Engagierte Unternehmen können unsere Arbeit auch mit einer Spende unterstützen. 
 
+### Privatpersonen
 
-{{< section >}}
-{{< headline-icon icon="icon-touch.svg" >}}
+Interessierte Menschen landen wir herzlich ein, unsere Projekte zu durchstöbern, zu verwenden, und weiterzuentwickeln. Wir freuen uns auch sehr über Spenden. Falls du überlegst, dich auch zu engagieren, wirf einen Blick auf unsere Community.
 
-{{< headline-brackets-h2 color="blue"  >}}
-Unsere gesellschaftlichen Ziele
+{{< headline-brackets-h2 color="red" >}}
+Jetzt die Befreiung von Daten unterstützen!
+{{</ headline-brackets-h2 >}}
 
-{{< /headline-brackets-h2  >}}
+-   Offene Daten, in maschinenlesbarem Format, zur Verfügung stellen.
 
-{{< home/block-goals >}}
+-   Langfristige Fördergelder für die Erschließung Offener Daten bereitstellen.
 
-{{< button-center href="/ziele" >}}
-Ziele
-{{< /button-center >}}
-{{< /section >}}
+-   Anwendungen aus Offenen Daten in die eigene Arbeit einbinden und weiterentwickeln.
+
+-   Eigene Expertise in unserer Community teilen.
 
+-   Code for Germany mit einer Spende unterstützen
 
-{{< section >}}
-{{< headline-icon icon="icon-power.svg" >}}
+{{</ paragraph-center >}}
```

Comment:
Ist es möglich, dass dieser ganze Absatz zum Thema "Jetzt die Befreiung von Daten unterstützen!" in eine landing-callout box kommt? Ich glaube nicht, dass es so einfach ist, aber probiere es mal unten. 

```
<div class="landing-callout">

**Jetzt die Befreiung von Daten unterstützen!**

Diese Maßnahmen verhelfen uns zu mehr Open Data: 
-   Offene Daten, in maschinenlesbarem Format, zur Verfügung stellen.
-   Langfristige Fördergelder für die Erschließung Offener Daten bereitstellen.
-   Anwendungen aus Offenen Daten in die eigene Arbeit einbinden und weiterentwickeln, z.B. unsere <a href="/projekte">Projekte</a>.
-   Eigene Expertise in unserer <a href="/community">Community</a> teilen.
-   Code for Germany mit einer <a href="/spende">Spende</a> unterstützen.

</div>
```